File: app/src/main/java/com/talktiva/pilot/model/Links.java
package com.talktiva.pilot.model;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

public class Links implements Serializable {

    @SerializedName("next")
    private String next;

    @SerializedName("prev")
    private String prev;

    public String getNext() {
        return next;
    }

    public void setNext(String next) {
        this.next = next;
    }

    public String getPrev() {
        return prev;
    }

    public void setPrev(String prev) {
        this.prev = prev;
    }
}

File: app/src/main/java/com/talktiva/pilot/model/Links.kt
package com.talktiva.pilot.model

import com.google.gson.annotations.SerializedName

import java.io.Serializable

class Links : Serializable {

    @SerializedName("next")
    var next: String? = null

    @SerializedName("prev")
    var prev: String? = null
}
